signac/contrib/project.py

```diff
@@ -1220,7 +1230,7 @@ def repair(self, fn_statepoints=None, index=None, job_ids=None):
         for job_id in job_ids:
             try:
                 # First, check if we can look up the state point.
-                sp = self.get_statepoint(job_id)
+                sp = self.open_job(id=job_id).sp
```

Comment:
Similar case to above -- `open_job` validates the state point. May need to rethink this case.

Reply:
Can we use `read_statepoints()[job_id]` here?